src/main/java/org/apache/sysml/api/python/SystemML.py

```diff
@@ -59,6 +313,39 @@ def __init__(self, sc, *args):
             self.ml = sc._jvm.org.apache.sysml.api.MLContext(sc._jsc, monitorPerformance, setForcedSparkExecType)
         except Py4JError:
             traceback.print_exc()
+            
+    def start(self, sqlCtx):
+		try:
+			prevGlobalMLContext = MLContext.globalMLContext
+		except AttributeError:
+			pass
+		self.numTabs = 0
+		self.sqlCtx = sqlCtx
```

Comment:
Good point. I will update my PR. For some reason `self.sqlCtx = SQLContext.getOrCreate(sc)` is throwing an error. Instead am creating `self.sqlCtx = SQLContext(sc)` for now and we can replace it with cleaner call.